examples/cfg/cfg.json

```diff
@@ -1,17 +1,17 @@
 {
-  "control": {},
-  "scheduler": {},
-  "plugins": {
-    "collector": {
-      "cinder": {
-        "all": {
-          "endpoint": "https://public.fuel.local:5000",
-          "user": "admin",
-          "password": "admin"
+  "control": {
+    "plugins": {
+      "collector": {
+        "cinder": {
+          "all": {
+            "endpoint": "http://localhost:5000/v2",
```

Comment:
I've put `localhost` to make sure this example works out of the box on any controller node.
Readme is pretty clear about that config value
> "endpoint" - URL for OpenStack Identity endpoint aka Keystone (ex. "http://keystone.public.org:5000")
But I agree the example is rather uninformative. In nova plugin I've explicitly named this option `"openstack_auth_url": "http://localhost:5000/v2.0/"`.

Perhaps best option is to unify configuration fields' names across all OpenStack plugins while supporting old ones with deprecation warning.